src/parser/MetricCalculator.ts

```diff
@@ -92,6 +89,6 @@ export class MetricCalculator {
             metricResults.set(result.metricName, result);
         }
 
-        return [formatPrintPath(sourceFile.filePath, this.#config), metricResults];
```

Comment:
`this.#config` is unused now
=> delete completely?

Reply:
Right